src/hooks/useComment.jsx

```diff
@@ -0,0 +1,80 @@
+import { useParams } from 'react-router-dom';
+import { useQueryClient, useQuery, useMutation } from '@tanstack/react-query';
+
+import {
+  deleteComment as remove,
+  getCommentList,
+  postComment as post,
+  editComment as edit,
+} from '../apis/comment.js';
+
+export default function useComment() {
+  const { postId } = useParams();
+  const queryClient = useQueryClient();
+
+  const { data: commentList } = useQuery({
+    queryKey: ['comments', postId],
+    queryFn: () => getCommentList(postId),
+    staleTime: 1000 * 60,
+    onError: (error) => {
+      console.error('댓글을 불러올 수 없습니다.', error);
+    },
+  });
+
+  const postComment = useMutation({
+    mutationFn: ({ content, parentId }) => post({ postId, parentId, content }),
+    onSuccess: () => {
+      queryClient.invalidateQueries(['comments', postId]);
+    },
+    onError: (error) => {
+      const ERROR_STATUS = error.response.status;
```

Comment:
ERROR_STATUS 의 경우 error.response.status 값에 따라 value 가 변경될 수 있는 값이라 상수값 표기법인 UPPER_SNAKE_CASE 표기법이 아닌 일반 변수 표기법인 lowerCamelCase로 표기하는 것이 어떨까요?

```suggestion
      const errorStatus = error.response.status;
```